Library/Formula/stack.rb

```diff
@@ -0,0 +1,10 @@
+class Stack < Formula
+  desc "The Haskell Tool Stack"
+  homepage "https://www.stackage.org/"
+  url "https://github.com/commercialhaskell/stack/releases/download/v0.1.2.0/stack-0.1.2.0-x86_64-osx.gz"
```

Comment:
You should submit this to [Homebrew/binary](https://github.com/Homebrew/homebrew-binary) if we can’t build it from source. We’ll be able to move it in the core in the next version if the bug is fixed.